fastestimator/util/util.py

```diff
@@ -135,3 +177,30 @@ def strip_suffix(target: Optional[str], suffix: Optional[str]) -> Optional[str]:
     if target[-s_len:] == suffix:
         return target[:-s_len]
     return target
+
+
+def per_replica_to_global(data):
```

Comment:
does this belong in util or backend?

Reply:
util, because it is only used by tensorflow.

backend should contain things worked for both tf and torch

Reply:
also, Typing

Reply:
🥇

Reply:
k